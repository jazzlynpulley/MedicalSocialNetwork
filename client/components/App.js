//client/components/App.js
import React, { Component } from 'react';
import '../css/App.css';
class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h1 className="App-title">Jazzlyn Pulley, Steven Thompson, Brandon O'Brien, Cory Petersen</h1>
        </header>

      <body>
	<h1> Bug reporting coming soon </h1>
	<h1> <a href="https://github.com/jazzlynpulley/MedicalSocialNetwork"> For source code, visit our github</a> </h1>
      </body>
     </div>
     );
  }
}
export default App;
