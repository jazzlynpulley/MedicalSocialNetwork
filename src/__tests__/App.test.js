import React from 'react';
import App from '../../client/components/App.js';
import renderer from 'react-test-renderer';

test('app homepage renders correctly', () => {
  const component = renderer.create(
    <App page="localhost:8000">Medical Social Network</App>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
