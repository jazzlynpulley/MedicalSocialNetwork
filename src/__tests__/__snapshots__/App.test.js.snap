// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Jazzlyn Pulley, Steven Thompson, Brandon O'Brien, Cory Petersen
    </h1>
  </header>
  <body>
    <h1>
       Bug reporting coming soon 
    </h1>
    <h1>
       
      <a
        href="https://github.com/jazzlynpulley/MedicalSocialNetwork"
      >
         For source code, visit our github
      </a>
       
    </h1>
  </body>
</div>
`;
